server/README.md

```diff
@@ -23,11 +23,11 @@ oc login -u system:admin
 ```
 oc edit scc restricted
 
-change: 
+change:
```

Comment:
Most of these changes are from my editor config not allowing trailing whitespace.